tests/unit/features/partitions/executor/test_organize.py

```diff
@@ -43,6 +43,19 @@
             "expected_message": None,
             "expected_overwrite": None,
         },
+        # Files that should have the same name in two different partitions where one is
+        # a namespaced partition
```

Comment:
Are we testing an error scenario for this? (e.g. organizing to non-existing namespace, or directly to namespace without partition name)